### PR TITLE
Add  two desired_capabilities that can set retryBackoffTime and maxRe…

### PR DESCRIPTION
**Pull request by sunnyyukaige**
…tryCount in flutter driver, the relative code change and Readme update have been merged to flutter driver master branch. so update Appium document.

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/writing-running-appium/caps.md [hunk 1]
```diff
@@ -280,3 +282,11 @@ Driver](/docs/en/drivers/ios-uiautomation.md).
 
 ### WinAppDriver Only
 (For WinAppDriver specific capabilities, please refer to the documentation on the [Appium Windows Driver repo](https://github.com/appium/appium-windows-driver#windowsdriver-specific-capabilities) itself.)
+
+### Flutter driver only
```
Comment (KazuCocoa) [inline docs/en/writing-running-appium/caps.md:286]: it is better to manage capabilities on their own repositories like https://github.com/appium/appium-xcuitest-driver#desired-capabilities , I believe

**Review by KazuCocoa — APPROVED** [review]

File: docs/en/writing-running-appium/caps.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ These Capabilities span multiple drivers.
 |----|-----------|-------|
 |`settings[settingsKey]`| Update [Appium Settings](https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/settings.md) on session creation. |e.g., `'settings[mjpegScalingFactor]': 10`, `'settings[shouldUseCompactResponses]': true`|
 
+
+
```
Comment (KazuCocoa) [inline docs/en/writing-running-appium/caps.md:68]: and here are redundant new lines

**Review by KazuCocoa — CHANGES_REQUESTED** [review]

File: docs/en/writing-running-appium/caps.md (hunk 2 above)
Comment (sunnyyukaige) [reply docs/en/writing-running-appium/caps.md:68]: @KazuCocoa thanks for so quickly feedback, i  have update the caps document. 😋

Comment (KazuCocoa) [reply docs/en/writing-running-appium/caps.md:68]: Could you remove L67 and 68's new lines, too?

Comment (sunnyyukaige) [reply docs/en/writing-running-appium/caps.md]: updated, thanks

**Review by KazuCocoa — APPROVED** [review]
thanks :)

**Pull request merged**